sites/all/themes/orwell/css/orwell.main.css

```diff
@@ -2654,4 +2652,122 @@ html.js input.form-autocomplete {
   text-transform: uppercase;
 }
 
+.spotlight-box {
+  margin: -20px;
+}
+
+.spotlight-box__top__primary,
+.spotlight-box__top__secondary,
+.spotlight-box__middle__item,
+.spotlight-box__bottom__item {
+  padding: 20px;
+}
+
+.spotlight-box__top {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+}
+
+@media all and (min-width: 49rem) {
+  .spotlight-box__top .spotlight-box__top__primary {
+    -ms-flex-preferred-size: 66%;
+    flex-basis: 66%;
```

Comment:
you could use the sass function percentage(2/3) to get the exact value unless you intentionally wants the 1% off that 33% and 66% gives you